test/integration/base.py

```diff
@@ -1,50 +1,46 @@
-import unittest
-
-from .pelikan_client import PelikanClient
-from .pelikan_server import PelikanServer
-
-SERVER_PORT = 12345
-ADMIN_PORT = 12346
+from client import DataClient, AdminClient
```

Comment:
Ah I didn't know that- should have researched why . was prefixed.

Shall we request python3 in the script?